test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -80,28 +92,38 @@ func TestPackages(t *testing.T) {
 		flavors = append(flavors, flavor)
 	}
 	for _, f := range flavors {
-		for _, test := range packagesTestsWithSkippedFlavors {
-			flavor := f // capture range variable for parallel tests closure
-			if shouldSkip(test.skippedFlavors, flavor) {
-				continue
-			}
-			suite := test.t(flavor, flavor.Architecture)
-			t.Run(suite.Name(), func(t *testing.T) {
-				t.Parallel()
-				// FIXME: Fedora currently has DNS issues
-				if flavor.Flavor == e2eos.Fedora {
-					flake.Mark(t)
+		for _, method := range supportedInstallMethods {
+			for _, test := range packagesTestsWithSkippedFlavors {
+				flavor := f // capture range variable for parallel tests closure
+				if shouldSkipFlavor(test.skippedFlavors, flavor) {
+					continue
 				}
-				opts := []awshost.ProvisionerOption{
-					awshost.WithEC2InstanceOptions(ec2.WithOSArch(flavor, flavor.Architecture)),
-					awshost.WithoutAgent(),
+				if shouldSkipInstallMethod(test.skippedInstallationMethods, method) {
+					continue
 				}
-				opts = append(opts, suite.ProvisionerOptions()...)
-				e2e.Run(t, suite,
-					e2e.WithProvisioner(awshost.Provisioner(opts...)),
-					e2e.WithStackName(suite.Name()),
-				)
-			})
+				// TODO: remove once ansible+suse is fully supported
+				if flavor.Flavor == e2eos.Suse && method == installMethodAnsible {
```

Comment:
This will be done in a following PR to isolate the different fixes needed for the ansible script, and have some e2e tests running as soon as possible